src/Video/Renderer.hpp

```diff
@@ -19,138 +19,158 @@ namespace Video {
     namespace Geometry {
         class Geometry3D;
         class Rectangle;
-    }
-    
+    } // namespace Geometry
+
     /// Handles rendering using OpenGL.
     class Renderer {
         public:
-            /// Create new renderer.
-            VIDEO_API Renderer();
-            
-            /// Destructor.
-            VIDEO_API ~Renderer();
-
-            /// Prepare for depth rendering static meshes.
-            /**
-             * @param viewMatrix The camera's view matrix.
-             * @param projectionMatrix The camera's projection matrix.
-             */
-            VIDEO_API void PrepareStaticMeshDepthRendering(const glm::mat4& viewMatrix, const glm::mat4& projectionMatrix);
+        /// Create new renderer.
+        VIDEO_API Renderer();
 
-            /// Render a static mesh.
-            /**
-             * @param geometry The geometry to render.
-             * @param viewMatrix The camera's view matrix.
-             * @param projectionMatrix The camera's projection matrix.
-             * @param modelMatrix Model matrix.
-             */
-            VIDEO_API void DepthRenderStaticMesh(Geometry::Geometry3D* geometry, const glm::mat4& viewMatrix, const glm::mat4& projectionMatrix, const glm::mat4& modelMatrix);
-            
-            /// Start rendering the frame.
-            /**
-             * @param renderSurface %RenderSurface to render to.
-             */
-            VIDEO_API void StartRendering(RenderSurface* renderSurface);
-            
-            /// Prepare for rendering static meshes.
-            /**
+        /// Destructor.
+        VIDEO_API ~Renderer();
+
+        ////Prepare for shadow renderin
```

Comment:
One slash too much. Some general grammatic mistakes in documentation.

Reply:
Fixed